### PR TITLE
BE: updated externalAPI

### PR DESCRIPTION
**Pull request by Integraluminium**
* new env variable to disable AI
* added await

### CODE REVIEW

File: backend/src/externalAPI/externalAPI.service.ts [hunk 1]
```diff
@@ -78,21 +77,23 @@ export class ExternalAPIService {
     userID: string,
   ): Promise<string> {
     try {
-      this.checkAllowed(userID);
+      await this.checkAllowed(userID);
       const paramsCheck = await this.checkParams(prompt, groupID);
       if (paramsCheck) {
+        console.log('REQUEST WOLFRAM', prompt);
```
Comment (Jozys) [inline backend/src/externalAPI/externalAPI.service.ts:83]: Consider using NestJS's logger instead the default console.log() statement.

The logger can be used like this
```typescript
 Logger.log('message', 'context');
```
The context in this case would be just "ExternalAPI" and the message "REQUEST WOLFRAM"

**Review by Jozys — CHANGES_REQUESTED** [review]
Looks fine in general. There is one aspect you might consider looking at.

File: backend/src/externalAPI/externalAPI.service.ts (hunk 1 above)
Comment (Integraluminium) [reply backend/src/externalAPI/externalAPI.service.ts]: Fixed in commit e075875

**Review by MichalskyL — APPROVED** [review]

**Review by Jozys — APPROVED** [review]
Looks fine.

**Pull request merged**